packages/react/scroll-area/package.json

```diff
@@ -25,10 +25,12 @@
     "@radix-ui/react-polymorphic": "workspace:*",
     "@radix-ui/react-primitive": "workspace:*",
     "@radix-ui/react-use-callback-ref": "workspace:*",
-    "@radix-ui/react-use-layout-effect": "workspace:*"
+    "@radix-ui/react-use-direction": "workspace:*",
+    "lodash.debounce": "^4.0.8"
```

Comment:
Need some suggestions here. Is there a smaller debounce package? Maybe we should use a react hook debounce lib from the wild because this will not `clearTimeout` if component unmounts mid-debounce.

Reply:
Curious on what we decide, throttle and debounce utils would be useful to have on hand.

Reply:
I've written my own custom one inline for now. When we need it elsewhere we can abstract it into its own package I guess 🙂 

https://github.com/radix-ui/primitives/pull/624/files#diff-839f9a54400069c816ef65581a175e1b45f4b13fca70ab5c0e6a6d9174ab9d34R895